sources/packages/forms/src/form-definitions/approvedenydesignation.json

```diff
@@ -0,0 +1,1390 @@
+                        "when": null,
+                        "eq": ""
+                    },
+                    "overlay": {
+                        "style": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "properties": {},
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "tag": "p",
+                    "id": "ed1rim"
+                },
+                {
+                    "label": "Designation Locations",
+                    "labelPosition": "top",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "description": "",
+                    "tooltip": "",
+                    "disableAddingRemovingRows": true,
+                    "conditionalAddButton": "",
+                    "reorder": false,
+                    "addAnother": "",
+                    "addAnotherPosition": "bottom",
+                    "layoutFixed": false,
+                    "enableRowGroups": false,
+                    "initEmpty": false,
+                    "customClass": "locations-grid",
+                    "tabindex": "",
+                    "hidden": false,
+                    "hideLabel": true,
+                    "autofocus": false,
+                    "disabled": false,
+                    "tableView": false,
+                    "modalEdit": false,
+                    "defaultValue": [
+                        {
+                            "approved": false
+                        }
+                    ],
+                    "persistent": true,
+                    "protected": false,
+                    "dbIndex": false,
+                    "encrypted": false,
+                    "redrawOn": "data",
+                    "clearOnHide": true,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "allowCalculateOverride": false,
+                    "validateOn": "change",
+                    "validate": {
+                        "required": false,
+                        "minLength": "",
+                        "maxLength": "",
+                        "customMessage": "",
+                        "custom": "",
+                        "customPrivate": false,
+                        "json": "",
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                    },
+                    "errorLabel": "",
+                    "errors": "",
+                    "key": "locationsDesignations",
```

Comment:
Do we have a validation to ensure that we have at least on checkbox selected?

Reply:
Added validation.